File: Ladder Pattern/ladder_pattern.java
import java.util.Scanner;

public class ladder_pattern {


    public static void Print_Ladder()
    {
        int num;
        Scanner input = new Scanner(System.in);
        System.out.print("How many ladder do you want to print: ");
        num = input.nextInt();

        for(int i=0;i<=num;i++)
        {
            System.out.println("*    *");
            System.out.println("*    *");

            if(i<num)
            {
                System.out.println("******");
            }
        }


    }

    public static void main(String[] args) {

        // Problem is if n = 2;
        // to hume ladder print karni hai
        // *    * 
        // *    *
        // ******
        // *    * 
        // *    *
        // ******
        // *    * 
        // *    *

        Print_Ladder();



    }
}

File: Ladder Pattern/ladder_pattern.cpp
#include <iostream>
using namespace std;


void PrintLadder()
{
    int num;
    printf("How many ladder you want to print: ");
    scanf("%d",&num);

    int i=0;
    while (i<=num)
    {
        cout<<"*    *"<<endl;
        cout<<"*    *"<<endl;

        if(i<num)
        {
            cout<<"******"<<endl;
        }
        i++;
    }
}

int main(){

    // Problem is if n = 2;
    // to hume ladder print karni hai
    // *    * 
    // *    *
    // ******
    // *    * 
    // *    *
    // ******
    // *    * 
    // *    *

    PrintLadder();
     

    

    return 0;
}